dependencies/lib-magic_enum/getter/CMakeLists.txt.in

```diff
@@ -0,0 +1,25 @@
+# \file dependencies/lib-magic_enum/getter/CMakeLists.txt.in
+# Configured by @CMAKE_CURRENT_LIST_FILE@
+#
+# @ONLY variables:
+#  * CMAKE_GENERATOR - which generator to use
+#  * SRC_DIR - where to extract the archive
+#  * BUILD_DIR - where the integrated sources are built
+
+cmake_minimum_required(VERSION 3.1)
+
+project(getter NONE)
+
+include(ExternalProject)
+externalproject_add(get-sol2
```

Comment:
Shouldn't this target have a different name?

Reply:
Oops. Just fixed.